src/palace/manager/feed/annotator/base.py

```diff
@@ -236,9 +236,7 @@ def content(cls, work: Work | None) -> str:
                 and work.summary.representation
                 and work.summary.representation.content
             ):
-                content = work.summary.representation.content
-                if isinstance(content, bytes):
-                    content = content.decode("utf-8")
+                content = work.summary.representation.content.decode("utf-8")
```

Comment:
The typing changes here made it clear that `work.summary.representation.content` will always be `bytes`